src/Evolution/Systems/RadiationTransport/M1Grey/BoundaryConditions/DirichletAnalytic.cpp

```diff
@@ -0,0 +1,242 @@
+// Distributed under the MIT License.
```

Comment:
This is just moving things into a cpp file without functionality changes, correct?

Reply:
Correct